src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/tests/latest/test_storage_blob_scenarios.py

```diff
@@ -279,6 +279,42 @@ def test_storage_blob_container_operations(self, resource_group, storage_account
         self.storage_cmd('storage container exists -n {}', account_info, c) \
             .assert_with_checks(JMESPathCheck('exists', False))
 
+    @api_version_constraint(ResourceType.MGMT_STORAGE, min_api='2017-07-29')
```

Comment:
You don't need that. As long as the test is in the `latest` folder, it will guarantee to run under latest profile.